qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -14,3 +14,9 @@
                 - memory_protection_key:
                     cpu_model_flags += ",+pku,check"
                     flags = "pku ospke"
+        - amd:
+            vendor_id = 'amd'
+            variants:
+                - flag_1:
```

Comment:
Why `flag_1`? Is there other type?

Reply:
none for now, I didn't find a good name...., will check again.

Reply:
@nanliu-r You don't have to specify a variant if no other type.

Reply:
sorry, I misunderstand, there is the other type, but that is not a series with this.